Library/Homebrew/.simplecov

```diff
@@ -12,9 +14,21 @@ SimpleCov.start do
   # tests to be dropped. This causes random fluctuations in test coverage.
   merge_timeout 86400
 
+  at_fork do |pid|
+    # This needs a unique name so it won't be ovewritten
+    command_name "#{SimpleCov.command_name} (#{pid})"
+
+    # be quiet, the parent process will be in charge of output and checking coverage totals
+    print_error_status = false
+  end
+
   if ENV["HOMEBREW_INTEGRATION_TEST"]
+    # This needs a unique name so it won't be ovewritten
     command_name "#{ENV["HOMEBREW_INTEGRATION_TEST"]} (#{$PROCESS_ID})"
 
+    # be quiet, the parent process will be in charge of output and checking coverage totals
+    print_error_status = false
```

Comment:
@MikeMcQuaid, shouldn't this be `self.print_error_status = `?

Reply:
it's inside the `SimpleCov.start` so I don't believe so.